README.md

```diff
@@ -119,8 +119,8 @@ Generate a new Rails application. We recommend the latest Rails 5.0 or 4.2 relea
 
 ```
 # If you don't already have Rails at your disposal...
-gem install rails -v 5.0.0.1
-rails new my_app -m https://raw.githubusercontent.com/projecthydra/sufia/master/template.rb
+gem install rails -v 5.0.1
+rails new my_app -m https://raw.githubusercontent.com/projecthydra/sufia/v7.3.0.rc1/template.rb
```

Comment:
This change will require that we tweak the 7.3.0.rc1 tag to point to this latest commit. Is that acceptable?